src/test/java/org/jenkinsci/plugins/github_branch_source/EventsTest.java

```diff
@@ -188,7 +191,11 @@ private GHSubscriberEvent createEvent(String eventPayloadFile) throws IOExceptio
   private void waitAndAssertReceived(boolean received) throws InterruptedException {
     long watermark = SCMEvents.getWatermark();
     // event will be fired by subscriber at some point
-    SCMEvents.awaitOne(watermark, 1200, TimeUnit.MILLISECONDS);
+    SCMEvents.awaitOne(watermark, received ? 20 : 200, TimeUnit.MILLISECONDS);
```

Comment:
Before changes await was 1200 because of delay before test was 1 second, for now we leave 200 millis for cases when expected `received = false`